### PR TITLE
Pthreads Issue

### PR DESCRIPTION
**Pull request by DhairyaBahl**
@RReverser Kindly look into this PR and guide me if i have to improve it further.

Thanks :)

### CODE REVIEW

File: site/source/docs/porting/pthreads.rst [hunk 1]
```diff
@@ -4,6 +4,8 @@
 Pthreads support
 ==============================
 
+We are currently working on the shared memory support on COEP/COOP headers. It is a possibility that user's pthreads code will not work in deployed environment.For more information click `this <https://web.dev/coop-coep>`_
```
Comment (RReverser) [inline site/source/docs/porting/pthreads.rst:7]: This note is not correct. We (as in Emscripten) are not working on it. Instead, it's browsers that are shipping SharedArrayBuffer gated behind COOP/COEP, but then you also need to explain here what "COOP/COEP" abbreviations mean before linking out for details.

**Comment by DhairyaBahl** [comment]
@RReverser okay I ll rectify it :)

**Comment by DhairyaBahl** [comment]
@RReverser I've made few tweaks in the documentation. Kindly have a look at it and tell me if i have to make it better. 

Thanks :)

**Comment by DhairyaBahl** [comment]
@RReverser @sbc100 
This is the PR for issue #12472 . Kindly tell me if i have to make any improvements in this. 
Thanks :)

File: site/source/docs/porting/pthreads.rst [hunk 2]
```diff
@@ -4,6 +4,8 @@
 Pthreads support
 ==============================
 
+Browsers are currently shipping SharedArrayBuffer gated behind Cross Origin Opener Policy (COOP) and Cross Origin Embedder Policy (COEP) headers. It is a possibility that user's pthreads code will not work in deployed environment.For more information click `this <https://web.dev/coop-coep>`_
```
Comment (RReverser) [inline site/source/docs/porting/pthreads.rst:7]: Please expand on the second sentence - it's not really a possibility, they will definitely not work across browsers unless the headers are correctly set.

Might be worth wrapping this into a note or a warning or something like that too.

**Pull request closed without merging**